english/star-wars-spoiler/star-wars.md

```diff
@@ -1,3 +1,5 @@
 Bad idea. The internet destroyed you for even contemplating such a thing.
 
-[Go Back...](../marshmallow.md).
```

Comment:
Hi,

I'm having some trouble add the content without somehow deleting the last line of the file. I think it is because there is no blank line at the end of the document. See issue #2637.
(Issue https://github.com/udacity/create-your-own-adventure/issues/2637)

Can you please add a blank line at the end of the file?

Thanks,
Paul